ml-agents/tests/yamato/check_coverage_percent.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import print_function
+import sys
+import os
+
+SUMMARY_XML_FILENAME = "Summary.xml"
+
+
+def check_coverage(root_dir, min_percentage):
+    summary_xml = None
+    for dirpath, _, filenames in os.walk(root_dir):
+        if SUMMARY_XML_FILENAME in filenames:
+            summary_xml = os.path.join(dirpath, SUMMARY_XML_FILENAME)
+            break
+    if not summary_xml:
+        print("Couldn't find Summary.xml in root directory")
+        sys.exit(0)
+
+    with open(summary_xml) as f:
+        # Look for a line of the form
+        # <Linecoverage>73.9</Linecoverage>
+        lines = f.readlines()
+        for l in lines:
+            if "Linecoverage" in l:
+                pct = l.replace("<Linecoverage>", "").replace("</Linecoverage>", "")
```

Comment:
Should these tags or the string `Linecoverage` be static strings or are they guaranteed not to change and are okay yo hard code?

Reply:
I don't know about "guaranteed not to change" but they're not expected not to change very often. I'd add some more comments that this is quick-and-dirty